koku/masu/processor/orchestrator.py

```diff
@@ -306,7 +308,8 @@ def start_manifest_processing(  # noqa: C901
                 summary_task = summarize_reports.s(
                     manifest_list=manifest_list, ingress_report_uuid=self.ingress_report_uuid
                 ).set(queue=SUMMARY_QUEUE)
-                async_id = chord(report_tasks, group(summary_task, hcs_task))()
+                subs_task = collect_subs_report_data_from_manifest.s().set(queue=SUBS_EXTRACTION_QUEUE)
+                async_id = chord(report_tasks, group(summary_task, hcs_task, subs_task))()
```

Comment:
I'm wondering if we need to do this, or if this can be a scheduled task instead?

Reply:
I'm not sure I'm in support of a scheduled task here. I think having it be a part of the tasks triggered after a download handles initial ingestion of a new source better since we process multiple months here.